src/Apps/Artwork/Components/ArtworkBanner/__tests__/ArtworkBanner.test.tsx

```diff
@@ -22,17 +21,7 @@ describe("ArtworkBanner", () => {
           }
         }
       `,
-      mockResolvers: {
-        Artwork: () => ({
-          ...response,
-          // FIXME: Need to figure out how to return proper values for aliesed objects
-          context: (_source, _args, _context, info: GraphQLResolveInfo) => {
-            const alias = info.fieldNodes[0].alias!.value
-            const x = response[alias]
-            return x
-          },
-        }),
-      },
+      mockData: { artwork: response },
```

Comment:
👏

Reply:
Noice! ✨

Reply:
@alloy - unfortunately it still did not fix the issue with data with the same name for different contexts is not passed properly. Therefore there are still couple of line commented out in specs here. If you ever have time to look into that - would be awesome!

Reply:
Let's pair shortly tomorrow so I’m up to date on the current state.